lang/julia/Portfile

```diff
@@ -8,8 +8,8 @@ PortGroup           compilers 1.0
 compilers.choose    fc f77 f90
 compilers.setup     require_fortran -g95
 
-github.setup        JuliaLang julia 1.5.4 v
-revision            0
+github.setup        JuliaLang julia 1.6.0 v
+revision            1
```

Comment:
Please change this back to `revision 0`.

The way this works is that MacPorts upgrades look at `version_revision` and update if there’s a bump. In this case, we’ll be at release of version `1.6.0` for the first (zeroth) time, so set the revision to `0`.

If we foul something up or discover something later on that affects the version `1.6.0` install, then we fix it and just bump the revision.

Reply:
Thanks for your information. I don't quite understand how `revision` works before.